### DISCUSSION TITLE
Notification bell has a new message but when I click on it, there's nothing

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20561] [Other Feature Feedback, Questions, & Ideas]
This bug feels a bit annoying

**Comment by SilasRodrigues19** [comment]
I saw this too, it started to happen not long ago

**Comment by radar-blip** [comment]
I haven't seen this happen but if you're sure its a bug,
you can file a bug report here: 
https://support.github.com/contact/bug-report
